test/ts/test_utils/send_transactions.ts

```diff
@@ -26,8 +26,8 @@ function findMethod(abi: Web3.AbiDefinition[], name: string, inputTypes: string)
 // method of a contract.
 // Truffle contract instance cannot handle overloaded methods
 // (truffle will only handle the first implementation of the method).
-export default function sendTransaction(
-    truffleContractInstance: any,
+export function sendTransaction(
+    truffleContractInstance: Web3.ContractInstance,
```

Comment:
Is this supposed to be a truffle contract instance?  Or a web3 contract instance?  Those are two very different things -- we should make sure the key and value are consistent.

Reply:
@t4sk labeled this param as a truffle contract instance. To my understanding, it's a Web3 contract instance.